data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1845 @@
+# Copyright (c) 2022 by Arachi-Lover
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+mission "Heliarch Investigation 1"
+	name "Transport Heliarch Agents"
+	description "Bring <bunks> Heliarch agents to <destination>."
+	minor
+	passengers 3
+	to offer
+		random < 65
+		"coalition jobs" >= 70
+		has "Coalition: Contributor: done"
+		not "Resistance: Questions: done"
+	source
+		near "Quaru" 4 10
+	destination "Ring of Friendship"
+	on offer
+		conversation
+			`A group of Heliarch agents rapidly approach you as you step onto the spaceport, looking rather hasty as they stand in front of you.`
```

Comment:
Is 'hasty' the right word here? 'Flustered' or 'agitated' maybe?

Reply:
Changed to agitated.